### PR TITLE
Add awesomeWM icon (#1938)

### PR DESCRIPTION
**Pull request by phatbhoy67**
**Issue: #1938**

### Checklist
  - [x] I updated the JSON data in `_data/simple-icons.json`
  - [x] I optimized the icon with SVGO or SVGOMG
  - [x] The SVG `viewbox` is `0 0 24 24`

### Description

Isolated "a" from official source full wordmark [.svg](https://awesomewm.org/images/awesome-logo.svg) available from the [website](https://awesomewm.org) header to create [favicon](https://awesomewm.org/images/icons/aw_16.png) version of the icon.

Hex value `#535d6c` is from the [logo](https://awesomewm.org/images/awesome-logo.svg). 

Built on GitPod, working as expected.

### CODE REVIEW

**Comment by foo-dogsquared** [comment]
Neatly done, @phatbhoy67 ! 

Though, it should be **awesomeWM** instead of **awesomeVM** (with the WM meaning "windows manager"). 
(It also means you have to change the filename of the SVG.) 

After correcting that, this PR is ready for merging.

File: _data/simple-icons.json [hunk 1]
```diff
@@ -315,6 +315,11 @@
             "hex": "0099DD",
             "source": "https://www.aventrix.com/press"
         },
+        {
+            "title": "awesomeVM",
```
Comment (foo-dogsquared) [inline _data/simple-icons.json]:
```suggestion
            "title": "awesomeWM",
```

File: icons/awesomevm.svg [hunk 2]
```diff
@@ -0,0 +1 @@
+<svg role="img" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><title>awesomeVM icon</title><path d="M0 24V8.25h16.5V7.5H0V0h24v24h-7.5v-8.25h-9v.75h8.25V24z"/></svg>
```
Comment (foo-dogsquared) [inline icons/awesomevm.svg]:
```suggestion
<svg role="img" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><title>awesomeWM icon</title><path d="M0 24V8.25h16.5V7.5H0V0h24v24h-7.5v-8.25h-9v.75h8.25V24z"/></svg>
```

**Comment by phatbhoy67** [comment]
🔍 Great catch @foo-dogsquared  👏

**Comment by foo-dogsquared** [comment]
Great! Thanks for the quick update. :)

**Pull request merged**